Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py

```diff
@@ -1,29 +1,252 @@
+that allows you to capture Use Cases, Requirements and Inputs/Outputs.
+
+Example Design document for the this Integration (HelloWorld):
+https://docs.google.com/document/d/1wETtBEKg37PHNU8tYeB56M1LE314ux086z3HFeF_cX0
+
+
+HelloWorld API
+--------------
+
+The HelloWorld API is a simple API that shows a realistic use case for an XSOAR
+integration. It's actually a real API that is available to the following URL:
+https://soar.mastersofhack.com - if you need an API Key to test it out please
+reach out to your Cortex XSOAR contacts.
+
+This API has a few basic functions:
+- Alerts: the endpoint returns mocked alerts and allows you to search based on
+a number of parameters, such as state (ACTIVE or CLOSED), type, timestamp. It
+can also return a single alert by ID. This is used to create new Incidents in
+XSOAR by using the ``fetch-incidents`` command, which is by default invoked
+every minute.
+The alert endpoint also used in Playbooks to retrieve a specific alert by id,
+as well as changing the alert status to "CLOSED" once resolved.
+
+- Reputation (ip and domain): these endpoints return, for an IP and
+domain respectively, a WHOIS lookup of the entity as well as a reputation score
+(from 0 to 100) that is used to determine whether the entity is malicious. This
+endpoint is called by XSOAR reputation commands ``ip`` and ``domain`` that
+are run automatically every time an indicator is extracted in XSOAR. As a best
+practice of design, it is important to map and document the mapping between
+a score in the original API format (0 to 100 in this case) to a score in XSOAR
+format (0 to 3). More information: https://xsoar.pan.dev/docs/integrations/dbot
+
+- Scan: to demonstrate how to run commands that are not returning instant data,
+the API provides a scan endpoint that simulates scanning a host and generating
+a report after the scan is completed. The API has endpoints to start a scan,
+which returns a job ID, poll for the scan status and, if the scan is completed,
+retrieved the job results.
+This function is used in conjunction of the HelloWorld Scan playbook that uses
+the GenericPolling mechanism to implement the job polling loop. The results
+can be returned in JSON or attachment file format.
+Info on GenericPolling: https://xsoar.pan.dev/docs/playbooks/generic-polling
+
+Please check the HelloWorld Design Document referenced above for details about
+the raw API responsens as well as the design details for this integration.
+
+This integration also has a ``say-hello`` command for backward compatibility,
+that doesn't connect to an API and just returns a ``Hello {name}`` string,
+where name is the input value provided.
+
+
+Integration File Structure
+--------------------------
+
+An integration usually consists of the following parts:
+- Imports
+- Constants
+- Client Class
+- Helper Functions
+- Command Functions
+- Main Function
+- Entry Point
+
+
+Imports
+-------
+
```

Comment:
i would mention here our internal imports (commonserverpython, demistomock and commonserveruserpython)